### ISSUE TITLE
Lesson 2.2 - adding the button in design view messes up the form

### ISSUE THREAD

**Issue by benlhroberts** [petabridge/akka-bootcamp#329]
Hi. I've seen a post on the Gitter chat but thought I'd mention it here as well. When opening the Main form designer, it doesn't show the chart, which isn't a major issue, but if the form is altered in any way (like adding the button in lesson 2.2), all of the charting controls (chartArea1, legend1, series1) get deleted from the form and it no longer works. 

The easiest solution is just to copy the code from Main.Designer.cs in the Completed folder, but this was a bit confusing at first so thought it best to flag it here. Thanks.

**Comment by zvrastil** [comment]
Yes, the problem is that project is targeting .NET Core, but VS Designer does only work correctly with DataVisualization components in .NET Framework. If you really want to follow the instructions, you have to recreate the project as .NET Framework Windows Forms application. Then, the designer works correctly. 

Seems like Petabridge updated the projects to .NET Core without actually checking if the instructions still work when followed step by step.

**Comment by gavinjcooper** [comment]
I was able to fix the designer by replacing the [System.Windows.Forms.DataVisualization](https://www.nuget.org/packages/System.Windows.Forms.DataVisualization/) nuget package with [WinForms.DataVisualization](https://www.nuget.org/packages/WinForms.DataVisualization/).
